python_predict_client/README.md

```diff
@@ -0,0 +1,29 @@
+# Python Predict Client
+
+It is the independent Python predict client for TensorFlow Serving.
+
+## Installation
+
+Copy the proto files.
+
+```
+cp ../tensorflow_serving/apis/{model,predict,prediction_service}.proto .
+```
+
+Replce the import path.
+
+```
+sed -i "s/tensorflow_serving\/apis\///g" {predict,prediction_service}.proto
```

Comment:
Thanks for the comment. It's reasonable and we don't want to add more dependencies than Bazel. The native python client for TensorFlow Serving is always needed and we have this project to manage it in https://github.com/tobegit3hub/deep_recommend_system/tree/master/python_predict_client .

I'm going to close this pull-reqeust and re-open if we have the better way to do that 😃